streamalert/classifier/clients/firehose.py

```diff
@@ -61,7 +61,10 @@ class FirehoseClient:
     _ENABLED_LOGS = dict()
 
     def __init__(self, prefix, firehose_config=None, log_sources=None):
-        self._prefix = prefix
+        self._prefix = (
+            '{}_'.format(prefix)
+            if firehose_config and firehose_config.get('use_prefix') else ''
```

Comment:
Opinion:

```
'{}_'.format(prefix)
if firehose_config and firehose_config.get('use_prefix')
else ''
```

Putting else on newline makes it more clearly distinguished from the `if` statement

Reply:
👍